tinyms/text/transforms.py

```diff
@@ -16,6 +16,7 @@
 
 from . import _transform_ops
 from ._transform_ops import *
+from ..data import *
```

Comment:
There is no need to import * from data module, instead just import the specific class.